apps/blog_api/lib/blog_api/utils.ex

```
@@ -0,0 +1,26 @@
+defmodule BlogApi.Utils do
```

반드시 에러의 디테일을 클라이언트에게 보여줘야한다면,
에러를 이쁘게 보여주는 모듈이 있어도 괜찮습니다.
아닌 경우에는
"성공"하는 로직에 신경을 쓰고, 실패하는 경우는 그냥 crash 나게 냅두세요.
성공하는 로직에만 신경쓰면서 개발한다면 `case do` 문에 성공하는 케이스만 넣고,
실패하는 경우는 그냥 패턴 매칭 실패로 crash가 나게 만들 수 있습니다.
crash 나도 서버가 다운되지 않으니 상관없습니다.

---

넵 컨트롤러 구현시 그 부분에 대해서 계속 고민했습니다.

다음에는 성공하는 케이스 위주로 작성해 나가겠습니다.

Let it crash!